resources/lib/plugin_content.py

```diff
@@ -335,27 +335,29 @@ def alphabetletter(self):
             xbmcplugin.setResolvedUrl(handle=int(sys.argv[1]), succeeded=False, listitem=xbmcgui.ListItem())
         letter = self.params.get("letter", "").upper()
         jumpcmd = ""
-        if letter in ["A", "B", "C", "2"]:
-            jumpcmd = "2"
-        elif letter in ["D", "E", "F", "3"]:
-            jumpcmd = "3"
-        elif letter in ["G", "H", "I", "4"]:
-            jumpcmd = "4"
-        elif letter in ["J", "K", "L", "5"]:
-            jumpcmd = "5"
-        elif letter in ["M", "N", "O", "6"]:
-            jumpcmd = "6"
-        elif letter in ["P", "Q", "R", "S", "7"]:
-            jumpcmd = "7"
-        elif letter in ["T", "U", "V", "8"]:
-            jumpcmd = "8"
-        elif letter in ["W", "X", "Y", "Z", "9"]:
-            jumpcmd = "9"
+        if letter in ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]:
+            jumpcmd = "firstpage"
+        elif letter in ["A", "B", "C"]:
+            jumpcmd = "jumpsms2"
+        elif letter in ["D", "E", "F"]:
+            jumpcmd = "jumpsms3"
+        elif letter in ["G", "H", "I"]:
+            jumpcmd = "jumpsms4"
+        elif letter in ["J", "K", "L"]:
+            jumpcmd = "jumpsms5"
+        elif letter in ["M", "N", "O"]:
+            jumpcmd = "jumpsms6"
+        elif letter in ["P", "Q", "R", "S"]:
+            jumpcmd = "jumpsms7"
+        elif letter in ["T", "U", "V"]:
+            jumpcmd = "jumpsms8"
+        elif letter in ["W", "X", "Y", "Z"]:
+            jumpcmd = "jumpsms9"
         if jumpcmd:
             xbmc.executebuiltin("SetFocus(50)")
```

Comment:
@sualfred Work for me in Leia but need to move down in line 360, to prevent blinking as posted @Guilouz